### PR TITLE
Initializing cluster_node_alert count in gluster_integration sync

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
tendrl-bug-id: Tendrl/gluster-integration#589

Signed-off-by: GowthamShanmugam <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
Tendrl/monitoring-integration/issues/374

**Comment by GowthamShanmugam** [comment]
@shtripat @r0h4n please review

**Comment by GowthamShanmugam** [comment]
Tested

File: tendrl/gluster_integration/sds_sync/__init__.py [hunk 1]
```diff
@@ -79,6 +79,20 @@ def run(self):
                     {"message": "Failed to sync cluster network details"}
                 )
 
+        if NS.tendrl_context.integration_id:
+            # Initialize alert node alert count
+            try:
+                key = 'clusters/%s/nodes/%s/alert_counters' % (
```
Comment (r0h4n) [inline tendrl/gluster_integration/sds_sync/__init__.py]: use 
if NS.tendrl.objects.ClusterNodeAlertCounters(
                        node_id=NS.node_context.node_id,
                        integration_id=NS.tendrl_context.integration_id
                   ).exists()

Comment (GowthamShanmugam) [reply tendrl/gluster_integration/sds_sync/__init__.py]: i will change this

**Pull request merged**